Software.py

```diff
@@ -282,7 +281,7 @@ def on_modified_async(self, view):
         global PROJECT_DIR
         # get active data will create the file info if it doesn't exist
         active_data = PluginData.get_active_data(view)
-        if active_data is None:
+        if active_data is None or active_data.project is None:
```

Comment:
Will this exclude tracking files edited outside of projects?

Reply:
We do our best to create the Untitled project but I noticed during testing that we'll occasionally get a NoneType project. This is to just prevent error chains from causing bigger issues. And based on testing, we'll create the project the next keystroke or two.